### ISSUE TITLE
new module: table

### ISSUE THREAD

**Issue by dominikwilkowski** [govau/uikit#8]
# code

- [ ] add responsive variation with scroll and `max-width`
- [ ] add responsive variation like this pen: http://codepen.io/dominikwilkowski/pen/eNvwXG

# uikit 1

<img width="730" alt="screen shot 2017-02-16 at 9 27 24 am" src="https://cloud.githubusercontent.com/assets/1266923/22998277/254a2cd0-f42a-11e6-9453-ec474ded1ea1.png">

[Design guide](http://guides.service.gov.au/design-guide/components/tables/index.html#tables)

**Comment by steveworley** [comment]
Hey @dominikwilkowski I've created a PR #202 to address this.

**Comment by steveworley** [comment]
While implementing the responsive table styles I noticed that if the column header was too long it would break the display on smaller devices. I added a small change to the responsive styles that will cause the label to appear as a block level element on it's own row on mobiles.

**Issue**

![image](https://user-images.githubusercontent.com/1840912/27460622-7c9d7d34-57f8-11e7-98db-8d128c9d96ce.png)

**Fix**

![image](https://user-images.githubusercontent.com/1840912/27460642-a39cba9e-57f8-11e7-92b6-9323d84900ba.png)

This will likely have limitations on how long `th` should be.
